resources/function_help/json/geom_to_wkb

```diff
@@ -1,7 +1,7 @@
 {
   "name": "geom_to_wkb",
   "type": "function",
-  "description": "Returns the Well-Known Binary (WKB) representation of a geometry as a binary blob.",
+  "description": "Returns the Well-Known Binary (WKB) representation of a geometry (a binary value)",
```

Comment:
```suggestion
  "description": "Returns the Well-Known Binary (WKB) representation of a geometry as a binary value",
```

I would do it this way.
I think it's an important detail (not in parenthesis), it clarifies that it's in binary format (and not as a string, as e.g. pgadmin shows it in the web interface).

Reply:
But is it not evident that a Well-Known Binary representation is a binary value? Are we not insulting the readers by repeating it?

Reply:
I get where you are coming from.
What it can tell you is that QGIS passes it around as a binary value internally (and not hexified or base64 or whatever encoded string as e.g. pgadmin does when displaying it).

Reply:
If we place it within parentheses we should avoid insulting the knowledgeable reader. ;-)
I have updated - including the word "as".

Reply:
I'm -0 on the parenthesis, IMHO they actually *do* suggest that it's repeated information (and hence are insulting the reader).
My take on this is, the first part (WKB) refers to *content*, the second part (binary value) refers to the *data format* internally used in qgis and the expression engine.
Compare to `3` vs. `'3'` vs. `3.0 -> the *content* is the same, but the *data format* is different.
If this is stating the obvious and repeating information, we should rather drop it completely.

Reply:
> If this is stating the obvious and repeating information, we should rather drop it completely.

Dropping it entirely is fine.
If the representation had been something else than binary, I agree that it would be useful to specify the representation, but when it is binary - as one would expect, it should not be necessary to mention it.